### ISSUE TITLE
Changing Profile settings

### ISSUE THREAD

**Issue by Gernash** [TanninOne/modorganizer#904]
If you edit your Profiles and Untick Local Game settings then your fallout4.ini in that profile gets renamed to fallout2.ini_ 
If you go to your configurator you get the following error.
![image](https://cloud.githubusercontent.com/assets/13113353/17235498/da346380-5584-11e6-9d98-382c2a86dd95.png)

**Comment by GSDFan** [comment]
This is true and repeatable on my system to some extent. After I did this a couple of times the error went away.
I am not sure if this is the way it is supposed to work or this needs further development. It appears MO still reads the INI files from MY games folder.
In testing this I can offer the following observations:
MO is supposed to rename the file from Fallout4.ini to Fallout4.ini_ when this option toggled.
It appears that a new generic Fallout4.ini is created each time this is toggled off/on. When I put a custom ini file in there and change the setting all my edits are gone.
Renaming Fallout4.ini_ to Fallout4.ini in the profile folder allows the configurator to work as planned.
